t/random_seed.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python3
+"""
+# random_seed.py
+#
+# Test fio's random seed options.
+#
+# - make sure that randseed overrides randrepeat and allrandrepeat
+# - make sure that seeds differ across invocations when [all]randrepeat=0 and randseed is not set
+# - make sure that seeds are always the same when [all]randrepeat=1 and randseed is not set
+#
```

Comment:
I think you can also add the usage, examples and requirements similar to what t/readonly.py, t/strided.py and other python based scripts.

Otherwise looks good, all the tests passed.

Reply:
Done. Thanks.